<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassScheduleManagement</class>
 <widget class="QDialog" name="ClassScheduleManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>634</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Class Schedule Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ClassScheduleManagement {
        background-color: #fef6f0   ;
        border: 1px solid black;
        border-radius: 10%;
}</string>
  </property>
  <widget class="QTableWidget" name="scheduleTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>781</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView#scheduleTable {
    border: 1.5px solid #2c3e50;
    border-radius: 8px;
    background-color: #ffffff;
    font-family: 'Segoe UI', sans-serif;
    font-size: 11pt;
    color: #2c3e50;
    gridline-color: #ecf0f1;
    alternate-background-color: #f7f9fa;
    selection-background-color: #d6eaf8;
    selection-color: #1a252f;
}

QHeaderView::section {
    background-color: #2c3e50;
    color: white;
    font-weight: bold;
    font-size: 11pt;
    padding: 6px;
    border: none;
    font-family: 'Segoe UI', sans-serif;
}

QTableWidget QTableCornerButton::section {
    background-color: #2c3e50;
    border: none;
}

QTableWidget::item {
    padding: 8px 10px;
    font-size: 9pt;
}

QTableWidget::item:selected {
    background-color: #d6eaf8;
    color: #1a252f;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="columnCount">
    <number>7</number>
   </property>
   <property name="horizontalHeaderLabels" stdset="0">
    <stringlist>
     <string>ID</string>
     <string>Class Name</string>
     <string>Teacher ID</string>
     <string>Day</string>
     <string>Start Time</string>
     <string>End Time</string>
     <string>Room</string>
    </stringlist>
   </property>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="editScheduleButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#editScheduleButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
        color: #2c3e50 ;
}

QPushButton#editScheduleButton:hover {
    background-color: #ee5253;
        color: #2c3e50 ;
}

</string>
   </property>
   <property name="text">
    <string>Edit Schedule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addScheduleButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#addScheduleButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
        color: #2c3e50 ;
}

QPushButton#addScheduleButton:hover {
    background-color: #ee5253;
        color: #2c3e50 ;
}

</string>
   </property>
   <property name="text">
    <string>Add Schedule</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit#searchBar {
    color: black;;
    border: 1.5px solid black;;
    border-radius: 10px;
        font-size: 16px;
        padding: 0px 8px 0px 8px;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Search by Class Name </string>
   </property>
  </widget>
  <widget class="QPushButton" name="viewScheduleButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#viewScheduleButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
        color: #2c3e50 ;
}

QPushButton#viewScheduleButton:hover {
    background-color: #ee5253;
        color: #2c3e50 ;
}

</string>
   </property>
   <property name="text">
    <string>View Schedules</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeScheduleButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>590</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#removeScheduleButton {
    background-color: #ff6b6b;
    border: 1.5px solid black;
    border-radius: 10px;
    font-size: 16px;
        color: #2c3e50 ;
}

QPushButton#removeScheduleButton:hover {
    background-color: #ee5253;
        color: #2c3e50 ;
}

</string>
   </property>
   <property name="text">
    <string>Remove Schedule</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
